src/Azure.Functions.Cli/npm/lib/install.js

```diff
@@ -47,6 +47,14 @@ const proxy = process.env.npm_config_https_proxy ||
             process.env.HTTP_PROXY ||
             process.env.http_proxy;
 
+const telemetryInfo = os.EOL
+    + 'Telemetry' + os.EOL
+    + '---------' + os.EOL
+    + 'The Azure Functions Core tools collect usage data in order to help us improve your experience. ' + os.EOL
+    + 'The data is anonymous and doesn\'t include any user specific or personal information. The data is collected by Microsoft.' + os.EOL
+    + os.EOL
+    + 'You can opt-out of telemetry by setting the FUNCTIONS_CORE_TOOLS_TELEMETRY_OPTOUT environment variable to \'1\' or \'true\' using your favorite shell.' + os.EOL
```

Comment:
btw, we will need this for homebrew, chocolatey and apt. Homebrew is probably the easiest to update, but we should look into others too before we release

Reply:
Yes, i think I still have the PRs from our last attempt :). Will just update them

Reply:
@ahmelsayed / @fabiocav , I turned telemetry [off for now](https://github.com/Azure/azure-functions-core-tools/pull/1532/files#diff-cb827ae1b3f8f061cc7b979172f1223bR12-R13). 
I moved this out to a separate PR. Once we are ready to start the telemetry, we can merge these PRs below (will add another one later to turn it on) and turn it on. I would prefer to not block this PR.
- https://github.com/Azure/azure-functions-core-tools/pull/1555
- https://github.com/Azure/homebrew-functions/pull/15
- https://github.com/ahmelsayed/publish-script/pull/1